specification/dimensions/pricingunit.md

```diff
@@ -0,0 +1,31 @@
+# Pricing Unit
+
+The Pricing Unit represents a provider specified pricing measurement unit, which forms the foundation for determining unit prices of utilizing a particular resource and/or service. Common examples include the number of hours a compute appliance was running for a given period, the number of gigabytes * the number of hours (gigabyte-hours) for a storage appliance, or a cumulative count of requests for a network appliance or API service. For certain services providers can choose block pricing i.e., million requests as a base unit.
+PricingUnit is complementary to and describes how [QuantityInPricingUnit](#quantityinpricingunit) is measured.
+
+The PricingUnit column MUST be present in the billing data. This column MUST be of type String. It MUST NOT contain null if [QuantityInPricingUnit](#quantityinpricingunit) is not null. The PricingUnit value MUST match what is shown on the invoice if a pricing unit is present on the invoice.
+
+## Column ID
+
+PricingUnit
+
+## Display name
+
+Pricing Unit
+
+## Description
+
+Represents a provider specified pricing measurement unit, which forms the foundation for determining unit prices of utilizing a particular resource and/or service.
```

Comment:
```suggestion
A measurement unit specified by the provider, serving as the basis for calculating unit prices associated with using a specific resource and/or service.
```

Reply:
I would love to avoid the term "basis". It's jargon that most people don't understand based on my conversations with customers outside of Finance.

Reply:
As a non-finance+non-native I would say that _basis_ is as understandable as _foundation_ in this context. 
@flanakin / @udam-f2 - maybe we should ship version 1.0 and eventually rethink such wordings after wider community feedback?

Reply:
@udam-f2 I had a quick chat with @flanakin and Michael mentions that basis can be widely unknown in a non-finance world based on his past experience.
What do you think about such small tweak?
**_serving as the foundation (aka basis) for calculating unit prices_**

Reply:
I'm ok with foundation.